pkgs/applications/science/electronics/nanovna-saver/default.nix

```diff
@@ -2,19 +2,28 @@
   lib,
   python3,
   fetchFromGitHub,
+  fetchpatch,
   wrapQtAppsHook,
 }:
 python3.pkgs.buildPythonApplication rec {
   pname = "nanovna-saver";
-  version = "0.5.3";
+  version = "0.5.4";
 
   src = fetchFromGitHub {
     owner = "NanoVNA-Saver";
     repo = pname;
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-wKKjMcOx7NS2VAIk3OTAj7KWE1+CeAzctdgdidT+HMA=";
+    sha256 = "sha256-CLfgDQt2rOXtWwvEhlXEstPp28nFhuhiAPYL6EjZVu4=";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
Better add a link to the PR/issue, or a short explanation.

Reply:
I looked at the commit, and it seems a bit pointless. It just prevents the CHANGELOG to be installed as a data file. The commit is already on the main branch, so it will be in any next release. For me this outweights the need of having this in nixpkgs.
Can you please clarify if there's anything that I missed?

Reply:
Sorry for not clarifying this. Upstream removed the `CHANGELOG.md` file without removing it from `setup.py`. Without the patch, building will fail with this error:
```
error: can't copy 'CHANGELOG.md': doesn't exist or not a regular file
```
Link to issue: https://github.com/NanoVNA-Saver/nanovna-saver/issues/579
(Was fixed directly, without PR)

I've added a note about the patch as a comment.